app/src/main/java/com/github/h3lp3rs/h3lp/MainActivity.kt

```diff
@@ -18,7 +18,7 @@ class MainActivity : AppCompatActivity() {
     fun sendMessage(view: View) {
         val editText = findViewById<EditText>(R.id.mainName)
         val message = editText.text.toString()
-        val intent = Intent(this, GreetingActivity::class.java).apply {
+        val intent = Intent(this, MedicalCardAcivity::class.java).apply {
             putExtra(EXTRA_MESSAGE, message)
```

Comment:
We should consider renaming this extra to a better name if we actually use it in our app.